src/styles.css

```diff
@@ -60,15 +60,35 @@ body {
   color: #91caff;
 }
 
+
+/* BEGIN about HTML styling */
+
+/*
+Header elements which are the first child of the about content container shouldn't have the default Ant Design top
+margin
+ */
+.about-content h2:first-child, h3:first-child {
```

Comment:
why just h2 and h3, why not h1?

Reply:
was just doing what is used but you're right, should be all